lib/Perl/Critic/Policy/Subroutines/ProhibitAmbiguousFunctionCalls.pm

```diff
@@ -0,0 +1,169 @@
+package Perl::Critic::Policy::Subroutines::ProhibitAmbiguousFunctionCalls;
+
+use strict;
+use warnings;
+use Readonly;
```

Comment:
`Readonly` not a declared dep in `Makefile.PL`?  Or could this be a `use constant` instead?

Reply:
Yes, we can switch to constant: earlier versions of this module used Readonly in some other places. but now we are down to two scalars.